### DISCUSSION TITLE
Stripe integration in self-hosted mode

### DISCUSSION THREAD

**Discussion by JieLiuRiver** [saleor/saleor#14073] [Q&A]
I used the self-hosted mode. After deployment, I accessed the dashboard and tried to install the stripe app, but I found that its install button was grayed out and unclickable. We currently use http, not https.
I don't know what steps I should do to successfully integrate stripe into my self-hosted service.

<img width="916" alt="image" src="https://github.com/saleor/saleor/assets/17538999/e1818d37-65f8-4a19-8a65-80d5f7f0c7be">

**Comment by timuric** [comment]
@JieLiuRiver Appstore is only available in Saleor Cloud and on Prem Enterprise customers. You would have to host stripe your self, please take a look at the apps repository https://github.com/saleor/apps

**Reply by moigamijunior** [reply]
How do you host the Stripe itself? I've cloned and run the stripe app, and tried to install it via the dashboard, but I got the same error. How do I use the Stripe app in self-hosted Seleor? Is this possible?

**Reply by richardcoder** [reply]
Same problem here, I've clone and run the stripe app locally and installed it via the dashboard. But after I click into the stripe app installed, its configuration form cannot show.  
Any help appreciated.
<img width="959" alt="CA7E880D-155E-4b91-8C7D-33F24ED23186" src="https://github.com/saleor/saleor/assets/23336824/ce9510c3-14cc-4260-acff-d8955b7244ea">
<img width="862" alt="D97BA261-7289-4c26-A271-664794628A6A" src="https://github.com/saleor/saleor/assets/23336824/dbf84b25-d914-4fcf-9f54-9439b5f2da9a">

**Comment by ebarahona** [comment]
If you're hosting the app yourself make sure you have the correct environment variables in your app's .env file set (localhost with port or whatever domain you've assigned to it):

APP_IFRAME_BASE_URL=https://myapp.mydomain.com
APP_API_BASE_URL=https://myapp.maydomain.com

You can verify the app is running correctly but visiting:  https://myapp.mydomain.com/api/manifest
